docs/conf/sponsorship/prospectus.rst

```diff
@@ -0,0 +1,214 @@
+
+Our conferences are run by team of volunteers, and we work hard to keep ticket
+prices affordable for a broad range of attendees. Your sponsorship makes it
+possible for all sorts of documentarians to attend our events, whether they're a
+solo freelancer or they run a docs team for a tech giant. Becoming a sponsor
+demonstrates your commitment to and support of good documentation, and the
+people who build it.
+
+.. raw:: pdf
+
+   PageBreak
+
+Sponsorship Packages
+====================
+
+The following options are suggested sponsorship levels. We are happy to discuss
+adjustments and custom packages.
+
+Each of these packages applies to *one* conference, either USA or Europe. Ask us
+about discounts packages for sponsoring both conferences.
+
+First Draft
+--------------------
+
+The **First Draft** package is aimed at startups, non-profits and open source
+companies:
+
+-  One (1) tickets
+
+-  Small logo & link on the Write the Docs website
+
+-  Name included in press release and event mailings
+
+-  Display 1 promotional (“Swag”) item at the conference (provided by
+       sponsor)
+
+The **First Draft** package costs **US$750**
+
+
+Second Draft
+----------------
+
+The **Second Draft** package for when you are no longer a startup:
+
+-  Two (2) tickets
+
+-  Large logo & link on the Write the Docs website
+
+-  Name included in press release and event mailings
+
+-  Display 1 promotional (“Swag”) items at the conference (provided by
+       sponsor)
+
+The **Second Draft** package costs **US $1500**
+
+Publisher
+---------------------------
+
+The **Publisher** package highlights your company as a force in the industry and
+community:
+
+-  Five (5) tickets
+
+-  Full size logo & link on the Write the Docs website
+
+-  Name included in press release and event mailings
+
+-  Display 1 promotional (“Swag”) items at the conference (provided by
+       sponsor)
+
+The **Publisher** package costs **US $5000**
+
+
+.. raw:: pdf
+
+   PageBreak
+
+Other Sponsorship Opportunities
+===============================
+
+The following à la carte offerings are available to increase visibility
+at the event.
+
+Financial Aid Sponsorships
+--------------------------
+
+Write the Docs is always trying to make our community bigger and more
+inclusive. Talk to us about sponsoring travel and/or registration
+expenses for an attendee who wouldn’t otherwise be able to attend.
+
+Lanyard
+----------------
+
+Provide your branded lanyard for our name tags.
+
+Writing Day
+--------------------
+
+Sponsor the Writing Day on Sunday, where we get together to help improve the
+documentation of many projects.
+
+-  Host a sponsored workshop or Writing Day session
+
+Party
+--------------
+
+We all know that some of the most valuable networking takes place after hours.
+Sponsorship of the official party includes:
+
+-  Mention of company on schedule for the party, and an in-person thanks
+
+-  Food, Non-alcoholic and Alcoholic drink options
```

Comment:
not sure this is needed here (party implies food and drinks), and it's out of place in a list of "what you get for your sponsorship"